envergo/geodata/utils.py

```diff
@@ -192,6 +192,9 @@ def to_geojson(obj, geometry_field="geometry"):
 def get_data_from_coords(lng, lat, timeout=0.5, index="address"):
     url = f"https://data.geopf.fr/geocodage/reverse?lon={lng}&lat={lat}&index={index}&limit=1"  # noqa
 
+    if is_test():
```

Comment:
Mmm… Ça me chagrine un petit peu de devoir surcharger manuellement ces fonctions dans les tests. Ça me semble un peu laborieux et sans doute que ça va rajouter du travail.

Est-ce que tu crois que ce serait automatisable ? [Peut-être avec autouse ?](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request)

Reply:
@thibault Je ne suis pas certain de bien comprendre l'objection.
J'ai ajouté autouse, ce qui évite d'avoir à ajouter la fixture dans les arguments du test. Mais il faut quand même l'invoquer dans les imports.
L'exception lancée lorsque l'on est dans un runtime de test est un simple garde fou pour éviter que l'on écrive un nouveau test en oubliant d'utiliser la fixture. Je trouve ça assez pratique, facile à comprendre et à corriger. Mais je reste ouvert aux suggestions.

Reply:
Je pourrais aussi mettre le mock directement dans la method, mais je ne trouve pas ca trés élégant.
```suggestion
    if is_test():
        return { "some": "data"}
```